omero/developers/Web/WebclientPlugin.txt

```diff
@@ -267,14 +267,13 @@ are three things to consider:
 URLs in templates
 ^^^^^^^^^^^^^^^^^
 
-The `url` tag has `changed in Django 1.6
-<https://docs.djangoproject.com/en/dev/releases/1.5/#overview>`_ from
-`{% url myview %}` to `{% url 'myview' %}`. In order for your OMERO 4.4.x
-install to understand the new syntax, it is simply a matter of adding
-`{% load url from future %}` to the top of the template file. By using the
-new-style syntax and the `{% load url from future %}` the template
-will be valid in both OMERO 4.4.x and OMERO 5. The
-`{% load url from future %}` line does no harm when used on Django 1.6.
+The :djangodoc:`url tag <ref/templates/builtins/#url>` has changed in
+`Django 1.8`_ from `{% url myview %}` to `{% url 'myview' %}`. In order
```

Comment:
There is a mismatch as it says 1.6 but link point to 1.5. We have a lot of 4.4 references here. I wasn't sure what to do with them. This meant to clean up the links to point to the latest doc. @will-moore Once that PR is merged could you please review this section